tokens/psp22/5F9aiiwLMPC6fFxxwqHvJpm7h5T4Xm93mJT6cpDrQnKkLFoK/info.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Mr Hankey",
+  "symbol": "SHIT",
+  "type": "PSP22",
+  "contractAddress": "5F9aiiwLMPC6fFxxwqHvJpm7h5T4Xm93mJT6cpDrQnKkLFoK",
+  "decimals": 12,
+  "description": "Most famous meme $SHIT",
+  "website": "https://the-shit.xyz/",
+  "sourceCode": "",
+  "links": [
+    {
+      "name": "X/Twitter",
+      "url": "https://x.com/mr_hankey_shit"
+    },
+    {
+      "name": "Telegram",
+      "url": "https://t.me/mr_hankey_shit"
+    },
+    {
+      "name": "Discord",
+      "url": ""
+    }
+  ],
+  "customTags": [
+    "$SHIT Contract Address"
+  ],
+  "warnings": [],
+  "owner": "5DMCcKK5HuEtSQ9QT9eunuzLy7owhLGX5g77Es9fsoRyhcXj",
```

Comment:
According to my checks, the owner is a different account.

Reply:
Can you check once again please? I am sure address is correct according to INW2 data 

```json
{
   "_id":"6654e88f113b0a15f4ebbf50",
   "name":"Mr Hankey",
   "symbol":"SHIT",
   "decimal":12,
   "contractAddress":"5F9aiiwLMPC6fFxxwqHvJpm7h5T4Xm93mJT6cpDrQnKkLFoK",
   "tokenGeneratorContractAddress":"5CWKnYhJr7jFw7a2ojKhdL3iBBGYohkoSzQQnj5v7AjmDiWu",
   "creator":"5DMCcKK5HuEtSQ9QT9eunuzLy7owhLGX5g77Es9fsoRyhcXj",
   "mintTo":null,
   "tokenIconUrl":"QmVVwDAy9VVRfbhWqhRD4rrvUV2JDCEy5ShC9Ko7U1Vgj3",
   "totalSupply":21000000,
   "index":20,
   "isManagedByTokenGenerator":true,
   "createdTime":"2024-05-27T20:09:51.234Z",
   "updatedTime":"2024-07-16T21:42:31.279Z"
}
```

Reply:
Or owner should be INW2 contract address ?

Reply:
Owner is the owner -- the entity who can mint and do whatever they want to the contract. That's what the contract code indicates. If you didn't set-up the contract by yourself please ask for support whoever did it for you.

Reply:
@DamianStraszak I cant understand where did you get this information ? What owner's address you see? I am 100% sure its same as minter `5DMCcKK5HuEtSQ9QT9eunuzLy7owhLGX5g77Es9fsoRyhcXj`

Reply:
With all due respect, but you should not be asking me who the owner of your contract is, or how to get that information.

This form is to collect facts and notarize them, not to get you technical support.

Reply:
Okay thanks for support, will no more wasting time on Azero

Reply:
@DamianStraszak prove me i am wrong
<img width="1055" alt="Снимок экрана 2024-07-17 в 11 10 44" src="https://github.com/user-attachments/assets/e0872a5e-d044-4506-a76c-f19b9c96b303">

Reply:
This is being discussed with the InkWhale team now. Will let you know soon.

Reply:
Alright. The issue is that you provided incorrect metadata.json file. The owner is specified correctly, but I was getting incorrect results because the provided metadata did not match the contract. 

I think that the correct metadata file for your contract is https://github.com/InkWhale-net/contracts/blob/feature/upgrade-op-4/op4_contracts/contracts/psp22_standard/psp22_standard.json -- but I don't know that, it's only speculation. So I would encourage you to ask InkWhale to confirm that.

So, long story short -- the metadata.json needs to be corrected in the submission.

Reply:
@DamianStraszak got new information that may can help: When token was created and minted there was AZERO.ID under owner account, after creation/mint azero.id was removed, maybe this create somekind of missmatch under owner account ?

Reply:
@DamianStraszak yeah as we discussed previously metadata was just copy-pasted from other PR's and i took 'debug' one and didnt notice it. Updated now with 'release' one

Reply:
Alright. I checked the new version a everything is fine.